WhaleSpotting/react-app/src/components/SightingMapInfo.tsx

```diff
@@ -2,32 +2,64 @@ import React, { useEffect, useState } from "react";
 import WeatherApiModel from "../apiModels/WeatherApiModel";
 import { Chosen } from "./Map";
 import "../styles/SightingMapInfo.scss";
+import { Species, WhaleImageDictionary, WhaleVisualTextDictionary } from "../apiModels/CreateSightingApiModel";
 
 interface SightingMapInfoProps {
     chosen: Chosen | undefined;
 }
 
+interface IResponse {
+    lon: number | void,
+    lat: number | void,
+    species: Array<string>,
```

Comment:
If we do the changes in the backend, this will be
```suggestion
    species: Species[],
```

Reply:
Addressed.